rare/utils/utils.py

```diff
@@ -249,87 +251,27 @@ def get_size(b: int) -> str:
         b /= 1024
 
 
-def create_rare_desktop_link(type_of_link):
-    # Linux
-    if platform.system() == "Linux":
-        if type_of_link == "desktop":
-            path = os.path.expanduser("~/Desktop/")
-        elif type_of_link == "start_menu":
-            path = os.path.expanduser("~/.local/share/applications/")
-        else:
-            return
-        if not os.path.exists(path):
-            return
+def create_desktop_link(app_name=None, core: LegendaryCore = None, type_of_link="desktop", for_rare: bool = False) -> bool:
```

Comment:
It might be a bit nit-picky, `app_name` is guaranteed to be a string and maybe it should not be None, or even empty. Furthermore, do you think the `for_rare` argument should be combined with a dedicated value coming through the `app_name` argument? Or it wouldn't be as clean?

Reply:
I thought about using an enum here, but unfortunately Python's enums can't hold values. So I sticked with this.
No, I don't think we should repurpose the `app_name` argument to switch into "Rare mode" instead. That's way more prone to mistyping elsewhere.

Reply:
That's valid, keep it like this then.